src/Controller/EnMarche/TerritorialCouncil/PoliticalCommitteeController.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace App\Controller\EnMarche\TerritorialCouncil;
+
+use App\Controller\CanaryControllerTrait;
+use App\Entity\Adherent;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Security;
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+use Symfony\Component\Security\Core\User\UserInterface;
+
+/**
+ * @Route("/comite-politique", name="app_political_committee_")
+ *
+ * @Security("is_granted('POLITICAL_COMMITTEE_MEMBER')")
+ */
+class PoliticalCommitteeController extends Controller
+{
+    use CanaryControllerTrait;
+
+    /**
+     * @Route("", name="index", methods={"GET"})
+     *
+     * @param Adherent $adherent
+     */
+    public function indexAction(UserInterface $adherent): Response
+    {
+        $membership = $adherent->getPoliticalCommitteeMembership();
+        $politicalCommittee = $membership->getPoliticalCommittee();
+
+        if (!$politicalCommittee->isActive()) {
```

Comment:
what about doing this check in the voter for `POLITICAL_COMMITTEE_MEMBER` permission?